specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2022-06-01-preview/examples/FileValidate.json

```diff
@@ -1,13 +1,13 @@
 {
```

Comment:
We can remove this

Reply:
Yea we can remove from the examples. We'll need to change the spec too right??

Reply:
Removed it from examples and spec.